### PR TITLE
Titles redo 4

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _layouts/tags.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Documentation
+title: Support Information About CI/CD And Codeship
```
Comment (mlocher) [inline _layouts/tags.html]: ? Imo neither the old one nor the new one makes a lot of sense. Can we find a better title for the tag pages?

File: _pro/builds-and-configuration/cli.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Local CLI (Codeship Pro)
+title: Debugging And Setup With The Local CLI (Codeship Pro)
```
Comment (mlocher) [inline _pro/builds-and-configuration/cli.md]: _The Local CLI_ doesn't make a lot of sense imo. Could we get back to Marketing and ask for a real name. Could be the _Codeship Pro CLI_ or something else.

File: _pro/builds-and-configuration/image-registries.md [hunk 3]
```diff
@@ -1,12 +1,17 @@
 ---
-title: Using Image Registries
+title: Using Docker Image Registreis With Your CI/CD Builds
```
Comment (mlocher) [inline _pro/builds-and-configuration/image-registries.md]: typo with _Registries_

File: _pro/languages-frameworks/nodejs.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Node.js on Docker
+title: Using Node In CI/CD with Docker and Codeship Pro
```
Comment (mlocher) [inline _pro/languages-frameworks/nodejs.md]: Node.js :)

File: _pro/quickstart/codeship-configuration.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Project Setup Or Migration
+title: Setting Up Or Migrating A Project With Codeship Pro
```
Comment (mlocher) [inline _pro/quickstart/codeship-configuration.md]: _Migrating A Project to Codeship Pro_ doesn't make sense imo. Shouldn't that be _Migrating A Project **To** Codeship Pro_?

File: _layouts/tags.html (hunk 1 above)
Comment (ethangj) [inline _layouts/tags.html]: @mlocher pending: https://trello.com/c/1CuTVts6/228-investigate-dynamic-tags-page-titles

**Pull request merged**